lib/hanami/repository.rb

```diff
@@ -208,16 +209,33 @@ module ClassMethods
       #   end
       #
       #   UserRepository.adapter = adapter
+      # @since 0.5.0
+      # @api private
       def adapter=(adapter)
         @adapter = adapter
       end
 
-      # @since 0.5.0
```

Comment:
Good question. Actually I was thinking about removing/resetting all the ```@version``` comments as well for the ```InstanceMethods``` methods that were previous ```ClassMethods```

About removing the comments: it's still valid. Just not the preferred way anymore. But until we have decided what the new preferred way is, i'd leave the comments.